app/jobs/subscription_placement_job.rb

```diff
@@ -62,6 +62,12 @@ def cap_quantity_and_store_changes(order)
     unavailable_stock_lines_for(order).each do |line_item|
       changes[line_item.id] = changes[line_item.id] || line_item.quantity
       line_item.update(quantity: 0)
+
+      Spree::OrderInventory.new(order).verify(line_item, order.shipment)
+    end
+    if changes.present?
+      order.line_items.reload
+      order.update_order_fees!
```

Comment:
My concern around this job class is growing every time we touch it. Just look at this method's complexity. It's a lot for my little brain to process :see_no_evil: